### DISCUSSION TITLE
I didn't tested on windows but yes, it should work. You should see at least Power sensor.

### DISCUSSION THREAD

**Discussion by xczsonn** [cagnulein/qdomyos-zwift#350] [Q&A]
how can i setting?

I didn't tested on windows but yes, it should work. You should see at least Power sensor.
Let me know!
R.

_Originally posted by @cagnulein in https://github.com/cagnulein/qdomyos-zwift/issues/347#issuecomment-855386830_

**Comment by cagnulein** [comment]
just enable the ant+ power sensor on the setting of QZ and then use zwift with your dongle ant+ on your pc :)
